src/components/ParticipantInfo/VideoTrackPublicationInfo/VideoTrackPublicationInfo.tsx

```diff
@@ -50,8 +50,9 @@ export const VideoTrackInfo: React.FC<{
   return (
     <>
       <Datum label="Dimensions" value={getDimensionString(dimensions)} />
+      {/* Only show isSwitchedOff and trackSwitchOffReason for remote tracks */}
       {isSwitchedOff !== undefined && <Datum label="isSwitchedOff" value={isSwitchedOff} />}
-      <Datum label="isEnabled" value={isEnabled} />
+      {trackSwitchOffReason !== undefined && <Datum label="Switch Off Reason" value={trackSwitchOffReason} />}
       <Datum label="Bandwidth" value={trackBandwidth?.toLocaleString() + 'kbps'} />
```

Comment:
Same thing here as the AudioTrackPublicationInfo:
![Screen Shot 2022-08-11 at 4 22 21 PM](https://user-images.githubusercontent.com/77076398/184258699-9646e471-f0e6-4bd8-8624-90316a5cad2f.png)